examples/net_client/main.rs

```diff
@@ -1,50 +1,51 @@
 use amethyst::{
-    core::{frame_limiter::FrameRateLimitStrategy, SystemDesc, Time},
-    derive::SystemDesc,
-    ecs::{Join, Read, System, SystemData, World, WriteStorage},
-    network::*,
+    core::{frame_limiter::FrameRateLimitStrategy, Time},
+    ecs::{Read, System, Write},
+    network::simulation::{
+        laminar::{LaminarNetworkBundle, LaminarSocket},
+        NetworkSimulationResource, NetworkSimulationTime,
+    },
     prelude::*,
     utils::application_root_dir,
     Result,
 };
 use log::info;
 use std::time::Duration;
 
+// You'll likely want to use a type alias for any place you specify the NetworkResource<T> so
+// that, if changed, it will only need to be changed in one place.
+type NetworkResourceImpl = NetworkSimulationResource<LaminarSocket>;
+
 fn main() -> Result<()> {
     amethyst::start_logger(Default::default());
 
+    let socket = LaminarSocket::bind("0.0.0.0:3455").expect("Should bind.");
```

Comment:
Nitpick: don't use a dot at the end of `expect` messages :)
https://doc.rust-lang.org/src/core/result.rs.html#998

Rust puts a colon right after the message and debug-prints an error

Reply:
That's a good point. I didn't know that.